watertap/unit_models/zero_order/feed_zo.py

```diff
@@ -80,3 +114,59 @@ def load_feed_data_from_database(self, overwrite=False):
                 except KeyError:
                     _log.info(f"{self.name} component {j} was not defined in "
                               f"database water source. Value was not fixed.")
+
+        # Set initial values for mass flows in properties based on these
+        # Assuming density of 1000
+        for t in self.flowsheet().time:
+            for j in self.properties[t].params.solute_set:
+                self.properties[t].flow_mass_comp[j].set_value(
+                    value(self.flow_vol[t] * self.conc_mass_comp[t, j]))
+            self.properties[t].flow_mass_comp["H2O"].set_value(
+                value(self.flow_vol[t]*1000))
```

Comment:
Why are we hard coding the 1000 here? Can't we use the default density in the property model? The initial value isn't a big deal and this is close enough, just was curious.